LLK/scratch-vm
# Fix Comment import/save/load issue

### Resolves

Resolves issue found in smoke test on 08/24 where some comments weren't being imported correctly when testing the fix in #1522.

### Proposed Changes

Fix issue where block comments converted to workspace comments import correctly when there are no scripts on the workspace.

### Test Coverage

Added new integration test and fixture.

### How to Test this PR
1. Create a new 2.0 project.
2. Drag out a block, e.g. `move 10 steps`
3. Use the block context menu to add a block comment to the block.
4. Ensure that the project is saved.
5. Move the new block comment away from the block to detach it from the block.
6. Ensure that the project is saved.
7. Delete the original block, e.g. `move 10 steps`
8. Ensure that the project is saved.
9. Load this project in 3.0, the comment should exist as a workspace comment.
10. Download this project to your computer
11. Upload the project from your computer.
The project *should not* fail to load. The workspace comment *should* still be there.
